src/commands/transaction/payload.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ *
+ * Copyright 2018-present NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { command, metadata, option } from 'clime';
+import {
+    AggregateTransaction,
+    Convert,
+    CosignatureSignedTransaction,
+    NetworkType,
+    SignedTransaction,
+    Transaction,
+    TransactionMapping,
+    TransactionType,
+} from 'symbol-sdk';
+import { AnnounceTransactionsCommand } from '../../interfaces/announce.transactions.command';
+import { AnnounceTransactionsOptions } from '../../interfaces/announce.transactions.options';
+import { Profile } from '../../models/profile.model';
+import { OptionsResolver } from '../../options-resolver';
+import { CosignatureResolver } from '../../resolvers/cosignature.resolver';
+import { FormatterService } from '../../services/formatter.service';
+import { TransactionView } from '../../views/transactions/details/transaction.view';
+
+export class CommandOptions extends AnnounceTransactionsOptions {
+    @option({
+        flag: 'P',
+        description: 'Transaction payload to be announced.',
+    })
+    payload: string;
+
+    @option({
+        flag: 'c',
+        description: 'Cosignatures array to be added.',
```

Comment:
Can we provide an example? How are arrays specified in a parameter?

Reply:
Can you take a look at the documentation that I prepared for these changes?
https://www.notion.so/CLI-offline-txs-create-collect-announce-469a8d3c48bb42df898da73985bfcbb3